x-pack/plugins/cases/public/mocks.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CasesUiStart } from './types';
+
+const createStartContract = (): jest.Mocked<CasesUiStart> => ({
+  getAllCases: jest.fn(),
+  getAllCasesSelectorModal: jest.fn(),
+  getCaseView: jest.fn(),
+  getConfigureCases: jest.fn(),
```

Comment:
unless these are being used for mocks we just just leave them empty functions `()=>{}`

Reply:
I think is better to leave them as `jest.fn()` always as they are mocks.

Reply:
Yeah they'll be used as mocks for other plugins that leverage cases.